lib/response.js

```diff
@@ -868,13 +868,9 @@ res.cookie = function (name, value, options) {
     val = 's:' + sign(val, secret);
   }
 
-  if (opts.maxAge != null) {
-    var maxAge = opts.maxAge - 0
```

Comment:
Could you explain _why_ it's safe to remove the `- 0` part?

e.g. I'm quite sure that previously it was okay to pass `'123'` as maxAge, as `'123' - 0` is `123`, which then would work out to `Date.now() + 123`. But with the new code, that would be `Date.now() + '123'` which would be a string with "123" appended to the end...

Reply:
@LinusU haha, so that's it! I thought there was a reason for this but couldn't recall and was going to test it out to see at some point. You are definitely right about the behavior change! It seems like all the tests pass, so we probably could add a test around passing a string :thinking:

Reply:
Yeah, hehe 😄 

For Express 5 I think it would be great to throw on strings, and maybe start logging a deprecation in 4, because I think it's very unlikely that someone is intentionally passing a string here...

Reply:
> Could you explain _why_ it's safe to remove the `- 0` part?
> 
> e.g. I'm quite sure that previously it was okay to pass `'123'` as maxAge, as `'123' - 0` is `123`, which then would work out to `Date.now() + 123`. But with the new code, that would be `Date.now() + '123'` which would be a string with "123" appended to the end...

Thank you @LinusU for taking the time for such a detailed reply.  Actually currently we cannot pass the `maxAge` as a `String` and there is already a test case exits for this in **test/rest.cookie.js** , please check the attached screenshots regarding this. 


<img width="634" alt="Screenshot 2023-02-09 at 2 29 14 PM" src="https://user-images.githubusercontent.com/15016564/217766971-ef1b8fdd-ebf6-4d38-9cbb-3221afdd9f64.png">



<img width="393" alt="Screenshot 2023-02-09 at 2 25 54 PM" src="https://user-images.githubusercontent.com/15016564/217766940-f1bd672c-2d56-4f2a-bbf4-8f2c84c12f74.png">

Reply:
> @LinusU haha, so that's it! I thought there was a reason for this but couldn't recall and was going to test it out to see at some point. You are definitely right about the behavior change! It seems like all the tests pass, so we probably could add a test around passing a string 🤔

@dougwilson, currently we cannot pass the `maxAge` as a `String` and there is already a test case exits for this in **test/rest.cookie.js** , please check the above attached screenshots regarding this.

Reply:
@LinusU @dougwilson  I got the reason , there is a dependency package `cookie` ( https://github.com/jshttp/cookie/blob/master/index.js)  where  it is okay to pass '123' as maxAge, as '123' - 0 is 123.
So I think its good to keep our existing as it is. I am closing this pull request . Thank you so much for you time @LinusU  @dougwilson. 

Below is the screenshot of dependency `cookie` package ,where we are validating maxAge.
<img width="407" alt="Screenshot 2023-02-09 at 10 28 53 PM" src="https://user-images.githubusercontent.com/15016564/217884656-e60dc937-f565-4d57-a15c-c3cb65e9a506.png">